### ISSUE TITLE
Vagrant asks for password after "Running triggets after up" and "Running triggers before halt"

### ISSUE THREAD

**Issue by kmgalanakis** [Varying-Vagrant-Vagrants/VVV#1442]
When doing `vargrant up` of `vagrant halt`, after `Running triggers after up` and `Running triggers before halt` respectively, Vagrant asks me to type the vagrant password.

I'm using:
* VVV 2.2.0
* Vagrant 2.0.2
* Virtualbox 5.2.8

on Windows 10 x64.

**Comment by tomjn** [comment]
Can you show the exact output? It's possible it's vagrant hosts updater

Also can you confirm you are **not** using Powershell ISE but a standard elevated command prompt/elevated powershell?

**Comment by kmgalanakis** [comment]
Hi @tomjn 

This is the output I'm getting when I do `vagrant up`.

```
D:\VVV>vagrant up
-f was unexpected at this time.
__ __ __ __
\ V\ V\ V / Varying Vagrant Vagrants
 \_/\_/\_/  v2.2.0-

Docs:       https://varyingvagrantvagrants.org/
Contribute: https://github.com/varying-vagrant-vagrants/vvv
Dashboard:  http://vvv.test

Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: A newer version of the box 'ubuntu/trusty64' for provider 'virtualbox' is
==> default: available! You currently have version '20180227.0.1'. The latest is version
==> default: '20180302.0.0'. Run `vagrant box update` to update.
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
==> default: Machine booted and ready!
Got different reports about installed GuestAdditions version:
Virtualbox on your host claims:   4.3.36
VBoxService inside the vm claims: 5.2.8
Going on, assuming VBoxService is correct...
[default] GuestAdditions 5.2.8 running --- OK.
Got different reports about installed GuestAdditions version:
Virtualbox on your host claims:   4.3.36
VBoxService inside the vm claims: 5.2.8
Going on, assuming VBoxService is correct...
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default:
    default: Guest Additions Version: 4.3.36
    default: VirtualBox Version: 5.2
==> default: [vagrant-hostsupdater] Checking for host entries
==> default: [vagrant-hostsupdater] Writing the following entries to (C:/WINDOWS/system32/drivers/etc/hosts)
==> default: [vagrant-hostsupdater]   192.168.50.4  vvv  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  vvv.dev  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  vvv.test  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  vvv.local  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  vvv.localhost  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  local.wordpress.test  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  src.wordpress-develop.test  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  build.wordpress-develop.test  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  src.wordpress-develop.dev  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater]   192.168.50.4  build.wordpress-develop.dev  # VAGRANT: c0996c65c7b5ba7bfe39d0907ba26b27 (default) / ced91f6c-ecd2-4fad-b5e3-11a36ec2c863
==> default: [vagrant-hostsupdater] This operation requires administrative access. You may skip it by manually adding equivalent entries to the hosts file.
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /srv/www => D:/VVV/www
    default: /srv/log => D:/VVV/log
    default: /vagrant => D:/VVV
    default: /srv/config => D:/VVV/config
    default: /srv/database => D:/VVV/database
==> default: Machine already provisioned. Run `vagrant provision` or use the `--provision`
==> default: flag to force provisioning. Provisioners marked to run always will still run.
==> default: Running provisioner: shell...
    default: Running: inline script
    default:  * Stopping MariaDB database server mysqld
    default:    ...done.
    default:  * Starting MariaDB database server mysqld
    default:    ...done.
    default:  * Checking for corrupt, not cleanly closed and upgrade needing tables.
==> default: Running provisioner: shell...
    default: Running: inline script
    default:  * Restarting nginx nginx
    default:    ...done.
==> default: Running triggers after up...
vagrant@127.0.0.1's password:
Connection to 127.0.0.1 closed.
```

Two lines before the end I'm been asked my password where I insert `vagrant`.

I'm not using Powershell ISE, or at least I don't think I do. Initially I tested this with Conemu but then I opened a normal command prompt with administrator rights. Actually this is where I'm pasting my output from.

Thanks

**Comment by tomjn** [comment]
hmmm I'm unfamiliar with ConEmu, but glad that just pressing enter works for you

As an aside, does `vagrant ssh` also ask for a password?

> `-f was unexpected at this time.`

Does this happen for all vagrant commands? Even those ran outside of VVV? e.g. `vagrant --version` ran in the main D: drive folder?

**Comment by kmgalanakis** [comment]
Hey Tom, 

Another thing I noticed is that when I `vagrant up`, I need to press `Enter` three times when I get the password prompt to get away from it.

`vagrant ssh` doesn't ask for a password though.

Finally, I'm getting this 

> -f was unexpected at this time.

when I do `vagrant up`, `vagrant halt` or `vagrant status`. When I do `vagrant --version` I don't get it.

**Comment by kmgalanakis** [comment]
I concluded that this has to do with the installed version of Vagrant because the issue doesn't occur with Vagrant 2.0.1.

Once I install Vagrant 2.0.2, the issue is there again.

**Comment by tomjn** [comment]
hmmm did you provision the box with 2.0.1 originally?

**Comment by kmgalanakis** [comment]
For each of my tries, I was creating a new VM by doing `vagrant destroy` and `vagrant up`, so no need to provision I guess.

**Comment by exlame** [comment]
I got the same problem with Vagrant 2.0.2 and fixed it with 2.0.1 or 2.0.4.
- Windows 10 64x
- VirtualBox 5.2.6
- Vagrant 
    - 2.0.1 : Work
    - 2.0.2 : Doesn't work
    - 2.0.4 : Work 
- cmd.exe
- VVV master branch

**Comment by kmgalanakis** [comment]
I confirm that with Vagrant 2.0.4, the issue is no longer reproducible.

Thanks @exlame
